tests/model_forms/tests.py

```diff
@@ -1875,6 +1875,17 @@ def test_model_multiple_choice_field(self):
         with self.assertRaises(ValidationError):
             f.clean([c6.id])
 
+    def test_model_multiple_choice_field_validate_choices(self):
+        class CustomModelMultipleChoiceField(forms.ModelMultipleChoiceField):
+            def validate_choices(self, qs, field_name, values):
```

Comment:
Making use of the arguments, perhaps by asserting their values, may be useful.

Reply:
I am taking a look at this. New to contributing, can you elaborate on what you mean by asserting their values?

Reply:
There should be assertions for the values received in this function.
It gets used, but it would be good to validate that qs/queryset, field_name, and values are what we expect.

Reply:
I'm sorry for another question but what would we expect them to be. I have a limited knowledge of what this PR is doing, I understand that it moves some validation into another function which allows you to skip/bypass it. In this case a qs should be a QuerySet object, values should probably be a list, and a field_name a string. is it a matter of a couple `assertIsInstance`?

Reply:
Yes, basically.
But also the value could/should be checked/asserted.

`qs` is about to be renamed to `queryset` though.